plugin.php

```diff
@@ -584,12 +588,24 @@ function rest_handle_options_request( $response, $handler, $request ) {
 
 		foreach ( $endpoints as $endpoint ) {
 			$accept = array_merge( $accept, $endpoint['methods'] );
+
+			$methods = array_keys( $endpoint['methods'] );
+			$method = array_shift( $methods );
+
+			if ( empty( $body['request_args'][ $method ] ) ) {
+				$body['request_args'][ $method ] = $endpoint['args'];
+			}
+
+			if ( empty( $body['schema'] ) && ! empty( $endpoint['schema'] ) ) {
+				$body['schema'] = $endpoint['schema'];
+			}
 		}
 		break;
 	}
 	$accept = array_keys( $accept );
 
 	$response->header( 'Accept', implode( ', ', $accept ) );
+	$response->set_data( (object) $body );
```

Comment:
How would it be empty, per https://github.com/WP-API/WP-API/pull/1262/files#diff-55d3bf5e833c76ee6e3c2f4be15fa1cdR577 ?